osbs/build/build_requestv2.py

```diff
@@ -144,6 +146,14 @@ def render_resource_limits(self):
             resources['limits'] = limits
             self.template['spec']['resources'] = resources
 
+    def render_resource_requests(self):
```

Comment:
docstring

Reply:
done

Reply:
where?

Reply:
I believe you shouldn't use 3rd person. 

- The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

Ref https://www.python.org/dev/peps/pep-0257/#id16

Reply:
now